ragna/deploy/_api/database.py

```diff
@@ -140,9 +140,15 @@ def get_chats(session: Session, *, user: str) -> list[schemas.Chat]:
     return [
         _orm_to_schema_chat(chat)
         for chat in session.execute(
-            select(orm.Chat).where(orm.Chat.user_id == _get_user_id(session, user))
+            select(orm.Chat)  # type: ignore[attr-defined]
+            .options(
+                joinedload(orm.Chat.messages).joinedload(orm.Message.sources),
+                joinedload(orm.Chat.documents),
```

Comment:
Why are the first two chained together, i.e. separated by a dot `.`, while the latter is another input argument, i.e. separated by a comma `,`?

Reply:
Each first-level `joinedload` (comma-separated) makes joins on Message and Document from Chat.
The second-level `joinedload` (dot separated) makes a join from Message to Source.

That is because we have : 
```
1 Chat ➞ N Documents
1 Chat ➞ N Messages
        1 Message ➞ N Sources
```

Reply:
Got it, thanks!